move-vm-backend/tests/assets/move-projects/substrate_stdlib_hash/Move.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "substrate_stdlib_hash"
+version = "0.0.0"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/eigerco/move-stdlib", rev = "main" }
+SubstrateStdlib = { local = "/Users/karlomardesic/Workspace/substrate-stdlib" }
```

Comment:
Do we change this afterwards in another PR?

Reply:
uuhh, good catch, I'm gonna fix this as soon as I merge the other PR

Reply:
Done